PWGCF/FemtoDream/Core/femtoDreamV0SelectionK0Short.h

```diff
@@ -0,0 +1,772 @@
+  void setInvMassLimits(float lowLimit, float upLimit)
+  {
+    fInvMassLowLimit = lowLimit;
+    fInvMassUpLimit = upLimit;
+  }
+
+  /// Set limit for the kaon rejection on the invariant mass
+  /// \param lowLimit Lower limit for the invariant mass distribution
+  /// \param upLimit Upper limit for the invariant mass distribution
+  void setKaonInvMassLimits(float lowLimit, float upLimit)
+  {
+    fRejectKaon = true;
+    fInvMassKaonLowLimit = lowLimit;
+    fInvMassKaonUpLimit = upLimit;
+  }
+
+  void setnSigmaPIDOffsetTPC(float offsetTPC)
+  {
+    nSigmaPIDOffsetTPC = offsetTPC;
+  }
+
+  void setChildRejectNotPropagatedTracks(femto_dream_v0_selection::ChildTrackType child, bool reject)
+  {
+    if (child == femto_dream_v0_selection::kPosTrack) {
+      posDaughTrack.setRejectNotPropagatedTracks(reject);
+    } else if (child == femto_dream_v0_selection::kNegTrack) {
+      negDaughTrack.setRejectNotPropagatedTracks(reject);
+    }
+  }
+
+  void setChildnSigmaPIDOffset(femto_dream_v0_selection::ChildTrackType child, float offsetTPC, float offsetTOF)
+  {
+    if (child == femto_dream_v0_selection::kPosTrack) {
+      posDaughTrack.setnSigmaPIDOffset(offsetTPC, offsetTOF);
+    } else if (child == femto_dream_v0_selection::kNegTrack) {
+      negDaughTrack.setnSigmaPIDOffset(offsetTPC, offsetTOF);
+    }
+  }
+
+  void setIsMother(bool IsMother)
+  {
+    fMotherIsLambda = IsMother;
+  }
+
+  void setRejectLambda(bool reject)
+  {
+    fRejectLambda = reject;
+  }
+
+ private:
+  int nPtV0MinSel;
+  int nPtV0MaxSel;
+  int nEtaV0MaxSel;
+  int nDCAV0DaughMax;
+  int nCPAV0Min;
+  int nTranRadV0Min;
+  int nTranRadV0Max;
+  int nDecVtxMax;
+  float pTV0Min;
+  float pTV0Max;
+  float etaV0Max;
+  float dCAV0DaughMax;
+  float cPAV0Min;
+  float tranRadV0Min;
+  float tranRadV0Max;
+  float decVtxMax;
+
+  float fInvMassLowLimit;
+  float fInvMassUpLimit;
+
+  bool fRejectKaon;
+  bool fRejectLambda;
+  float fInvMassKaonLowLimit;
+  float fInvMassKaonUpLimit;
+
+  float nSigmaPIDOffsetTPC;
+
+  bool fMotherIsLambda;
+
+  FemtoDreamTrackSelection posDaughTrack;
+  FemtoDreamTrackSelection negDaughTrack;
+
+  static constexpr int kNv0Selection = 9;
+
+  static constexpr std::string_view kSelectionNames[kNv0Selection] = {
+    "Sign", "PtMin", "PtMax", "EtaMax", "DCAdaughMax", "CPAMin",
+    "TranRadMin", "TranRadMax", "DecVecMax"}; ///< Name of the different
+                                              ///< selections
+
+  static constexpr femtoDreamSelection::SelectionType
+    kSelectionTypes[kNv0Selection]{
+      femtoDreamSelection::kEqual,
+      femtoDreamSelection::kLowerLimit,
+      femtoDreamSelection::kUpperLimit,
+      femtoDreamSelection::kUpperLimit,
+      femtoDreamSelection::kUpperLimit,
+      femtoDreamSelection::kLowerLimit,
+      femtoDreamSelection::kLowerLimit,
+      femtoDreamSelection::kUpperLimit,
+      femtoDreamSelection::kUpperLimit}; ///< Map to match a variable with
```

Comment:
Please stick to the naming conventions and don't use the `k` prefix by default. It makes expressions like `kSelectionNames[kNv0Selection]` quite confusing, since the constants starting with `k` are usually reserved for sizes of static arrays.